### PR TITLE
onload_profiles: fix nginx profile to parser for -s option

### PR DESCRIPTION
**Pull request by ol-adubovoj**

### CODE REVIEW

**Review by ol-alexandra — APPROVED** [review]

File: scripts/onload_profiles/wait-old-cluster-exit.opf-fragment [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # ceased to exist
 # If references to old onload stacks are still being released - wait.
 
-if echo "$@" | perl -n -e 'if(/\s-s/) {exit 1}'; then
+if echo "$@" | grep -q -e '[[:space:]]-s'; then
```
Comment (ivatet-amd) [inline scripts/onload_profiles/wait-old-cluster-exit.opf-fragment:8]: I'm not familiar with the Nginx profile, please help me to clarify:

As-is, if the user provided `-s signal` to `nginx`, we meant to invoke `onload_stackdump`, but we didn't and needed to invert the condition.

If that's the case, may I suggest expanding the commit message and then let's merge it?

Comment (ivatet-amd) [reply scripts/onload_profiles/wait-old-cluster-exit.opf-fragment:8]: Also, `grep` looks neater than `perl`!

Comment (ol-alexandra) [reply scripts/onload_profiles/wait-old-cluster-exit.opf-fragment:8]: > As-is, if the user provided `-s signal` to `nginx`, we meant to invoke `onload_stackdump`, but we didn't and needed to invert the condition.

exactly

Comment (ol-adubovoj) [reply scripts/onload_profiles/wait-old-cluster-exit.opf-fragment:8]: I just expanded the commit-message.
Please check this out.

Comment (ol-adubovoj) [reply scripts/onload_profiles/wait-old-cluster-exit.opf-fragment:8]: @ivatet-amd are you happy with my extended commit message or can you suggest something else?

Comment (ivatet-amd) [reply scripts/onload_profiles/wait-old-cluster-exit.opf-fragment:8]: Awesome, thank you. 👍

**Review by ivatet-amd — APPROVED** [review]

**Pull request merged**